test/test_prelude.py

```diff
@@ -0,0 +1,90 @@
+# coding: utf-8
+
+from pathlib import Path
+import shutil
+import requests
+from zipfile import ZipFile
+import os
+import glob
+import nibabel as nib
+import numpy as np
+
+import pytest
+
+from shimmingtoolbox.unwrap import prelude
+
+
+class TestCore(object):
```

Comment:
Just to be sure, this only test the prelude funciton and not the wrapper and dimension checks that you have done previously. Would it be posisble to test these to?

Reply:
This is only for prelude, we are removing the unwrap_phase for now as we only have one unwrapper so we only need to test for prelude.